apps/easyocr/src/main.py

```
@@ -0,0 +1,124 @@
+          file_path.unlink(missing_ok=True)
+          try:
+            output = { 'imageSize': { 'width': width, 'height': height }, 'results': results }
+            return Response(json.dumps(output, default=json_serializer, ensure_ascii=False), 200)
+          except Exception as e:
+            log_error('Error parsing JSON output:', e)
+            return Response('server error parsing results', status=500)
+        except Exception as e:
+          file_path.unlink(missing_ok=True)
+          log_error('Error processing image:', e)
+          return Response('Error processing image', status=500)
+      except Exception as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)